newton/_src/viewer/viewer.py

```
@@ -927,6 +970,31 @@ def log_points(
         """
         pass
 
+    def log_gaussian(
+        self,
+        name: str,
```

Ruff (0.15.2)

[warning] Unused method argument: `name`

(ARG002)